### PR TITLE
Fix/redemption strats solidly

### PR DESCRIPTION
**Pull request by carlomazzaferro**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/chains/src/bsc/redemptionStrategies.ts [hunk 1]
```diff
@@ -100,6 +100,14 @@ const redemptionStrategies: { [token: string]: [RedemptionStrategyContract, stri
     RedemptionStrategyContract.UniswapLpTokenLiquidator,
     underlying(assets, assetSymbols.HAY),
   ],
+  [underlying(assets, assetSymbols["sAMM-HAY/BUSD"])]: [
+    RedemptionStrategyContract.UniswapLpTokenLiquidator,
+    underlying(assets, assetSymbols.HAY),
```
Comment (vminkov) [inline packages/chains/src/bsc/redemptionStrategies.ts:105]: why is the output token HAY here? for flash loan liquidations we will likely need to repay with a stable asset, like BUSD or WBNB

File: packages/chains/src/bsc/liquidation.ts [hunk 2]
```diff
@@ -8,7 +8,11 @@ const liquidationDefaults: LiquidationDefaults = {
...
   ASSET_SPECIFIC_ROUTER: {
     [underlying(assets, assetSymbols["asBNBx-WBNB"])]: "0xcF0feBd3f17CEf5b47b0cD257aCf6025c5BFf3b7",
-    [underlying(assets, assetSymbols.BNBx)]: "0xcF0feBd3f17CEf5b47b0cD257aCf6025c5BFf3b7", // ApeSwap router
+    [underlying(assets, assetSymbols["vAMM-HAY/ankrBNB"])]: "0xd4ae6eCA985340Dd434D38F470aCCce4DC78D109", // Thena Solidly Router
+    [underlying(assets, assetSymbols["sAMM-HAY/BUSD"])]: "0xd4ae6eCA985340Dd434D38F470aCCce4DC78D109",
+    [underlying(assets, assetSymbols["vAMM-ANKR/HAY"])]: "0xd4ae6eCA985340Dd434D38F470aCCce4DC78D109",
+    [underlying(assets, assetSymbols["vAMM-ANKR/ankrBNB"])]: "0xd4ae6eCA985340Dd434D38F470aCCce4DC78D109",
+    [underlying(assets, assetSymbols["sAMM-jBRL/BRZ"])]: "0xd4ae6eCA985340Dd434D38F470aCCce4DC78D109",
   },
   SUPPORTED_OUTPUT_CURRENCIES: [constants.AddressZero, ankrBNB, WBNB, HAY, BUSD],
```
[the remark this answers is not in the document]
Comment (carlomazzaferro) [reply packages/chains/src/bsc/liquidation.ts:17]: Correct, we are OK with keeping ankrBNB or HAY

**Review by vminkov — APPROVED** [review]

File: packages/chains/src/bsc/redemptionStrategies.ts [hunk 3]
```diff
@@ -100,7 +100,17 @@ const redemptionStrategies: { [token: string]: [RedemptionStrategyContract, stri
     RedemptionStrategyContract.UniswapLpTokenLiquidator,
     underlying(assets, assetSymbols.HAY),
   ],
+  [underlying(assets, assetSymbols["sAMM-HAY/BUSD"])]: [
+    RedemptionStrategyContract.UniswapLpTokenLiquidator,
+    underlying(assets, assetSymbols.BUSD),
+  ],
+  [underlying(assets, assetSymbols["vAMM-HAY/ankrBNB"])]: [
+    RedemptionStrategyContract.UniswapLpTokenLiquidator,
```
Comment (vminkov) [inline packages/chains/src/bsc/redemptionStrategies.ts:108]: are we sure these pairs can be liquidated with the `UniswapLpTokenLiquidator` ?

Comment (carlomazzaferro) [reply packages/chains/src/bsc/redemptionStrategies.ts:108]: Yeah, now that I realise it, no they cannot...

Comment (carlomazzaferro) [reply packages/chains/src/bsc/redemptionStrategies.ts:108]: https://github.com/Midas-Protocol/contracts/blob/development/contracts/test/liquidators/UniswapLikeLpTokenLiquidatorTest.t.sol

Comment (carlomazzaferro) [reply packages/chains/src/bsc/redemptionStrategies.ts:108]: They can actually

**Pull request merged**